v2/api/dbformysql/v1alpha1api20210501/flexible_server_types_gen.go

```diff
@@ -2380,6 +2653,132 @@ func (availability *HighAvailability_Status) AssignPropertiesToHighAvailabilityS
 	return nil
 }
 
+// Deprecated version of Identity. Use v1beta20210501.Identity instead
+type Identity struct {
+	Type                   *IdentityType      `json:"type,omitempty"`
+	UserAssignedIdentities map[string]v1.JSON `json:"userAssignedIdentities,omitempty"`
```

Comment:
I think this is readonyl and should be removed in our config.

Reply:
Done.